pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -200,6 +200,24 @@ stdenv.mkDerivation rec {
     '';
 
   passthru = {
+    # Note for maintainers: if there is a hotfix we need to add and we want to prevent a mass rebuild,
+    # we can add it here, and reference it in the modules or packages that depend on it.
+    # util-linux transitively depends on util-linuxMinimal, and making a change in this file rebuilds
+    # the world. This lets us decouple hotfixes requiring mass rebuilds from fixing issues for one
+    # package or module that needs it.
+    #
+    # This was originally needed for Kubernetes (#405952 and #407810).
+    #
+    # Of course, keep this in sync with staging merges if we do end up needing anything.
+    withPatches = finalPackage.overrideAttrs (prev: {
```

Comment:
This seems pretty overcomplicated for me for something that I don't remember having come up very much before…

Reply:
Do you think we should just do overrides in two places then? I was split between both approaches but could see merits to either because it's a oneoff.

Reply:
I think having the passthru is fine but I'd remove it again once we don't need it any more.

Reply:
OK, yeah, completely fair. People should not rely on this. I'll update the tracking issue and fix the comment to indicate that this to a temporary hack.